GitHub_Learning/Feature/Onboarding/View/OnboardingViewController.swift

```diff
@@ -68,17 +73,23 @@ extension OnboardingViewController: UICollectionViewDataSource, UICollectionView
     
     func scrollViewDidEndDecelerating(_ scrollView: UIScrollView) {
         let currentPageIndex = Int(scrollView.contentOffset.x / view.frame.width)
-        presenter.presentingPage(at: currentPageIndex)
+        presenter.selectedPage(at: currentPageIndex)
     }
     
 }
 
 extension OnboardingViewController: OnboardingView {
     
     func showPage(at index: Int) {
-        let indexPath = IndexPath(item: index, section: 0)
-        layout.pagesCollectionView.scrollToItem(at: indexPath, at: .centeredHorizontally, animated: true)
-        layout.pagesControl.currentPage = index
+        let currentScrollIndex = Int(layout.pagesCollectionView.contentOffset.x / view.frame.width)
```

Comment:
Opakuje se ti i v `scrollViewDidEndDecelerating`. Muzes to vytahnout jako computed property a pouzivat to na obou mistech. `currentScrollIndex` je dobre jmeno pro obe.